pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -1,4 +1,4 @@
-package sfu
+package streamallocator
```

Comment:
Please advise on placement of this. This is one level below 'sfu' directory and uses structs from that namespace. Does not sound ideal. While 'sfu' namespace does not use anything from here (except tests which I will point to and had to be changed to avoid circular dependency), feels like this should live at the level of 'sfu' at least?

Reply:
this is easy to change later if it becomes an issue

Reply:
Sounds good